tidy3d/web/cli/app.py

```diff
@@ -89,8 +90,13 @@ def auth(req):
         current_apikey = get_description()
         message = f"Current API key: [{current_apikey}]\n" if current_apikey else ""
         apikey = click.prompt(f"{message}Please enter your api key", type=str)
+    try:
+        resp = requests.get(
+            f"{Env.current.web_api_endpoint}/apikey", auth=auth, verify=Env.current.ssl_verify
+        )
+    except (requests.exceptions.SSLError, ssl.SSLError):
+        resp = requests.get(f"{Env.current.web_api_endpoint}/apikey", auth=auth, verify=False)
```

Comment:
Can this still be an issue @lei-flex ? It sounds to me like if verify fails it just goes ahead without verification anyway, which doesn't sound more secure.

Reply:
that's our current behavior of other webapi invoke.  let's just keep it as same as other right now. 
the long story is that we hit very painful issue at beginning on  a few of customers.  they have local SSL issues and can't connect to our services with the normal way.
so we provided the auto hack way to release user's pain part. 
When we set the verify is false, we already have the warning message to show on web. so I think it is fine.
if we actually want to block any unverifed call, let's just create another ticket to block by user's config.